OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -45,11 +49,16 @@ public class AssetBrowserLogic : ChromeLogic
 		IReadOnlyPackage currentPackage;
 		Sprite[] currentSprites;
 		IModel currentVoxel;
+		ISound currentSound;
+		ISoundFormat currentSoundFormat;
+		Stream currentAudioStream;
```

Comment:
`currentAudioStream` is unused

Reply:
Brought it back in one of the new commits.